samples/python/beam_search_causal_lm/beam_search_causal_lm.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import openvino_genai
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('model_dir')
+    parser.add_argument('prompts', nargs='+')
+    args = parser.parse_args()
+
+    device = 'CPU'  # GPU can be used as well
+    pipe = openvino_genai.LLMPipeline(args.model_dir, device)
+
+    config = pipe.get_generation_config()
```

Comment:
create new one instead of overriding current config

Reply:
Updated